src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -609,11 +613,25 @@ private CompletableFuture<Void> startGame(GameLaunchResponse gameLaunchMessage)
         .thenApply(adapterPort -> {
           fafServerAccessor.setPingIntervalSeconds(5);
           gameKilled = false;
-          try {
-            process = forgedAllianceService.startGameOnline(gameLaunchMessage,
-                adapterPort, localReplayPort, rehostRequested);
-          } catch (IOException e) {
-            throw new GameLaunchException("Could not start game", e, "game.start.couldNotStart");
+          if (isMatchmakerGame) {
+            leaderboardService.getActiveLeagueEntryForPlayer(playerService.getCurrentPlayer(), gameLaunchMessage.getLeaderboard()).thenAccept(leagueBeanOptional -> {
+              String divisionName = leagueBeanOptional.map(bean ->
+                      bean.getSubdivision().getDivision().getNameKey() + " " + bean.getSubdivision().getNameKey())
+                  .orElse("unlisted");
```

Comment:
are passing the nameKey or the name? Seems like this would have to go through i18n somewhere

Reply:
We're passing the name key here, so the game can then handle the localization.

Reply:
Also shouldn't they be different options then to make localization easier?